services/API-service/src/api/event/event.service.ts

```diff
@@ -501,6 +516,14 @@ export class EventService {
         WHEN event."forecastSeverity" = 0.7 THEN 'Medium warning'
         WHEN event."forecastSeverity" = 0.3 THEN 'Low warning'
         END as "alertClass"`,
+        `CASE
```

Comment:
same question. we can probably lose alertClass here right?

Reply:
[same answer](https://github.com/rodekruis/IBF-system/pull/2036#discussion_r1986956365)

Reply:
closing this convo in favour of the other one.